### PR TITLE
added 0 variance error

### LINKED ISSUE #33: Do Zero-Count genes need to be filtered?
Hi Tony and Thevaa,

> LPWC::corr.bestlag(my_data[49:58,], timepoints = timepoints, max.lag = 20, penalty = "high", iter = 10) 

... no problem, just like in the vignette

LPWC::corr.bestlag(my_data, timepoints = timepoints, max.lag = 20, penalty = "high", iter = 10)
**Error in lags[j] <- val[which.max(corr)] : replacement has length zero
In addition: Warning message:
In cor(data[i, ], data[j, ]) : the standard deviation is zero**

I assume this is because my file contains all 20k genes, many of which are likely zero always. I will filter these out soon and follow up. 

fwiw my entire script:

dataFile = 'Data/Sub_391_2_pool_22adb_hg19_SOX2_plus__foregut_endoderm__32a860ee85db9ac9/genes.no_mt.ec.tab'
my_data = read.table(dataFile, header=TRUE, row.names=1, sep = '\t')
my_data = my_data[,-ncol(my_data)] # Drop description
timepoints = 0:96
solution = LPWC::corr.bestlag(my_data, timepoints = timepoints, max.lag = 20, penalty = "high", iter = 10)
dist <- 1 - solution$corr
plot(hclust(dist))

### PR DESCRIPTION
**Pull request by thevaachandereng**
closed #33

### CODE REVIEW

**Comment by thevaachandereng** [comment]
I added the error in the r file. I will work on the vignette and test cases next.

**Comment by thevaachandereng** [comment]
Sorry, I was fixing it on my bus ride home and I pushed it so that I can switch pull request.

**Comment by thevaachandereng** [comment]
@agitter could you fix the conflicts?

**Comment by thevaachandereng** [comment]
@agitter I double checked it today and this looks good.

File: vignettes/LPWC.Rmd [hunk 1]
```diff
@@ -245,6 +245,7 @@ dist <- 1 - LPWC::corr.bestlag(simdata[11:20, ], timepoints = timepoints, max.la
 cutree(hclust(dist), k = 3)
 ```
 
+## Filtering genes
```
Comment (agitter) [inline vignettes/LPWC.Rmd]: This is the last piece needed before merging the pull request.  Can you please write a couple sentences to explain that LPWC works best when run with genes that are significantly differentially expressed over time and genes that do not vary are not allowed?  The user must filter them before running LPWC.  You can give a code snippet to show users how to filter 0 variance genes.

**Comment by thevaachandereng** [comment]
Let's submit a new version to CRAN after this pull request.

**Comment by agitter** [comment]
@thevaachandereng I made a few small changes to the vignette.  See if you think they are okay.

Does that example code work on `simdata`?  When I tested it, `index0` was set to an empty integer so the next line removed all of the genes.

**Comment by thevaachandereng** [comment]
@agitter I fixed the issue. I was thinking people should only run it when they see they get the error message. But this will fix it for all cases. Feel free to merge when you are ready.

**Pull request merged**